build/root/WORKSPACE

```diff
@@ -23,7 +23,8 @@ load("@io_k8s_repo_infra//:repos.bzl", repo_infra_configure = "configure", repo_
 #            'override_go_version': used to specify an alternate go version provided
 #                                   by kubernetes/repo-infra
 repo_infra_configure(
-    go_version = "1.15.10",
+    go_version = "1.15.11",
+    override_go_version = "1.15.11",
```

Comment:
this is new/interesting... required starting in 1.15.11?

Reply:
there are no new go_rules for the bazel version we are using so we need to do this, see slack for context: https://kubernetes.slack.com/archives/C2C40FMNF/p1617371964211500?thread_ts=1617365695.206200&cid=C2C40FMNF

Reply:
@liggitt -- Here's the original context: https://github.com/kubernetes/repo-infra/pull/202

I borrowed this pattern from rules_go so that we would be able to start supporting prerelease versions of go.
Soon afterwards, starting in [rules_go v0.25.0](https://github.com/bazelbuild/rules_go/releases/tag/v0.25.0), you can download any version of go from golang.org/dl, with needing to hardcode it.

That said, it would be non-trivial to bump the rules_go version.

And then soon after that, we removed bazel!
So this is one of the last times we should need to use this override functionality.

Reply:
no worries, mostly just wondered if it was really needed / intentional. and yes, yay to not needing to do this going forward